common/lib/xmodule/xmodule/util/adaptive_learning.py

```diff
@@ -169,8 +169,12 @@ def get_student(self, student_uid):
         """
         url = self.generate_student_url(student_uid)
         payload = {'key_type': 'uid'}
-        response = requests.get(url, headers=self.request_headers, data=payload)
-        students = json.loads(response.content)
+        try:
+            students = []
```

Comment:
@macornwell This should go into the [`except` block](https://github.com/open-craft/edx-platform/pull/69#discussion_r148495978).